### PR TITLE
notify-api-412b use black to enforce python coding style

### PR DESCRIPTION
**Pull request by terrazoon**
The only relevant changes are in Makefile, Pipfile, and setup.cfg.  However, everything a significant PR gets merged, the conflicts become horrible due to the format changes, so recreating from scratch in this PR.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 ago = "~=0.0.95"
+black = "==23.7.0"
```
Comment (ccostino) [inline Pipfile:8]: Is there any reason not to change this so we'll pick up minor updates?

```suggestion
black = "~=23.7.0"
```

File: setup.cfg [hunk 2]
```diff
@@ -2,11 +2,12 @@
...
-line_length=80
-indent='    '
-multi_line_output=3
-known_third_party=notifications_utils,notifications_python_client
-known_first_party=app,tests
-include_trailing_comma=True
-use_parentheses=True
+[tool:isort]
+profile = black
+multi_line_output = 3
```
Comment (ccostino) [inline setup.cfg:7]: I'm assuming the removal of some of the `isort` flags is because `black` will take care of those things going forward?

**Review by ccostino — COMMENTED** [review]
Thanks, @terrazoon!  I had a couple of questions and didn't see any adjustments to `Makefile` - was there something supposed to be included?

**Comment by terrazoon** [comment]
> 

Some kind of GitHub problem?  It was there when I pulled down.  I made a small change in the comments to force a refresh of the PR.  Makefile should have "Pipfile run black ." added and Pipfile should have "black = '==23.7.0'"

File: setup.cfg (hunk 2 above)
Comment (terrazoon) [reply setup.cfg:7]: yes this was the request from Steven, to eliminate conflicts between isort and black.  The black documentation suggests using the black profile, which eliminates the need for most of those other settings (when isort didn't have a black profile).

Comment (terrazoon) [reply setup.cfg:7]: black is very strict and opinionated and disagrees with how isort does something things by default.

File: Pipfile (hunk 1 above)
Comment (terrazoon) [reply Pipfile:8]: > Is there any reason not to change this so we'll pick up minor updates?

I did not change this and do not see it when I pull down from GitHub or look at the code for the PR branch.  Don't know what is going on with GitHub (or my eyes).

File: setup.cfg (hunk 2 above)
Comment (ccostino) [reply setup.cfg:7]: That makes two of us as far as `isort` is concerned... 😆 thank you!

File: Pipfile (hunk 1 above)
Comment (ccostino) [reply Pipfile:8]: Ah, this is just from having run `pipenv install` before - this suggestion adjusts the reference in the `Pipfile` to allow for minor or patch level updates, which Dependabot helps us manage.

Comment (ccostino) [reply Pipfile:8]: Actually, let's not worry about this right now - the dependencies overall will be tackled soon enough as it is!

**Review by ccostino — APPROVED** [review]
Thanks again, @terrazoon!

**Pull request merged**